### ISSUE TITLE
Update the way the bean templates are defined and code is generated

### ISSUE THREAD

**Issue by fyrewall77** [michapringle/codesorcerer#3]
** Allow classes to have any name (no Def) required
** Classes that are up for generation must be in a package called *.<whatever>.bbbtemplate
** All generated classes will generate classes with the same name to *.<whatever>
** This allows the consumer of the library to add an exclusion for the *.<whatever>.bbbtemplate package if desired

**Comment by michapringle** [comment]
For example, I would write two classes in my package ...bean, and include in there ...bean.bbbtemplate, and after running the tool, I would have this result (generated code in the generated dir of course)

```
.
├── bean
│   ├── Address.java
│   ├── bbbtemplate
│   │   ├── Address.java
│   │   └── Person.java
│   └── Person.java
```

**Comment by fyrewall77** [comment]
Ha... I think the original BeanBuilder had this feature (you requested it).
You can put the Defs in a package called def and it will genererate the
beans in the package above! :)

**Comment by fyrewall77** [comment]
Yes... the name of the class in the a.b.c.def package will be the name of
the generaged class in the a.b.c package.
